clients/client-python/scripts/generate_version.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import re
+import configparser
+import subprocess
+from datetime import datetime
+
+from gravitino.constants.version import Version, VERSION_INI
+from gravitino.exceptions.gravitino_runtime_exception import GravitinoRuntimeException
+
+VERSION_PATTERN = r"version\s*=\s*['\"]([^'\"]+)['\"]"
+
+
+def main():
+    with open("setup.py", "r", encoding="utf-8") as f:
```

Comment:
Will this "setup.py" relative path be an issue to find the file when we run the gradle command in different `current directory`? For example, like running the gradle command in the gravitino home directory versus running it in the python directory.

Reply:
Shall we use the absolute path here?

Reply:
Yeah, I forgot to use absolute path here, I've fixed it.